### DISCUSSION TITLE
Is there a command to launch a window floating?

### DISCUSSION THREAD

**Discussion by mrwsl** [bakkeby/dwm-flexipatch#204] [Q&A] [answered]
I have a statuscmd enabled bar and I want to show a rain radar gif centered and floating. Therefore I created a rule:
```
RULE(.title = "radar.gif - mpv", .isfloating = 1, .iscentered = 1)
```
The command launching mpv:
```
setsid -f "$TERMINAL" -e mpv -loop=inf "$radar"
```
The problem is that the window is called "mpv" for the first seconds. Is there a way to start a window centered and floating besides creating rules?

**Comment by bakkeby** [comment, marked as answer]
mpv is not particularly flexible when it comes to specifying the instance or class, but it does allow for the title to be set.

I would try with a rule like:
```c
RULE(.class = "mpv", .title = "radar", .isfloating = 1, .iscentered = 1)
```

and a command like this:
```c
setsid -f "$TERMINAL" -e mpv --title="radar" -loop=inf "$radar"
```

The title came through as "radar" when applyrules is called in my testing, so I think this should be fine.

**Reply by mrwsl** [reply]
Thanks, it's working!
